lib/types/README.md

```diff
@@ -0,0 +1,15 @@
+# @intellectia/types
+
+To install dependencies:
+
+```bash
+pnpm install
+```
+
+To run:
+
+```bash
+pnpm run dist/esm/index.js
```

Comment:
I don't think pnpm can run the js file. I understand this is due to bun ( in that env it can ) . `node dist/esm/index.js` use this one instead. Thanks